app/scripts/components/academy/academy-modal.components.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Lifespan from 'lifespan';
+
+import {academyTutorialLabel} from '../../helpers/joyride.helpers.js';
+import Button from '../shared/button.components.jsx';
+import LocalClient from '~/stores/local-client.stores.jsx';
+
+export default class AcademyModal extends React.Component {
+	constructor(props) {
+		super(props);
+	}
+
+	async componentWillMount() {
+		this.client = LocalClient.instance();
+		this.lifespan = new Lifespan();
+
+		this.client.getStore('/prototypoStore', this.lifespan);
+	}
+
+	componentWillUnmount() {
+		this.lifespan.release();
+	}
+
+	static contextTypes = {
+		router: React.PropTypes.object.isRequired,
+	};
+
+	exit() {
+		this.client.dispatchAction('/store-value', {uiJoyrideTutorialValue: academyTutorialLabel});
```

Comment:
I forgot about this but can't we dispatch only once to store the different values?